### PR TITLE
Updating the cert-manager kustomization

### PR DESCRIPTION
**Pull request by simardeep1792**
Changes:
- Removed Istio Injection
- Changed the minimum pod resources for the deployements
- ~~Changed the images to Chaingaurd~~ (See the [comment](https://github.com/PHACDataHub/cpho-phase2/pull/178#issuecomment-1875800179) )
- Refactored the patches

### CODE REVIEW

File: k8s/cert-manager/crds/kustomization.yaml [hunk 1]
```diff
@@ -100,82 +116,27 @@ patches:
...
+
+  # # enable istio injection for the namespace
+  # - target:
+  #     kind: Namespace
+  #     name: cert-manager
+  #   # use ~1 instead of / in path (see https://github.com/kubernetes-sigs/kustomize/issues/1256)
+  #   patch: |-
+  #     - op: add
+  #       path: /metadata/labels/istio.io~1rev
+  #       value: asm-managed
```
Comment (Stephen-ONeil) [inline k8s/cert-manager/crds/kustomization.yaml:142]: Do you think there's a chance we might find some reason to re-enable istio injection in `cert-manager`? If not, I'd recommend deleting this block just so someone skimming the file in the future isn't confused.

**Review by Stephen-ONeil — APPROVED** [review]
One minor comment, but lgtm!

File: k8s/cert-manager/crds/kustomization.yaml (hunk 1 above)
Comment (simardeep1792) [reply k8s/cert-manager/crds/kustomization.yaml:142]: No problem I can delete it, Thanks :)

**Review by vedantthapa — CHANGES_REQUESTED** [review]

File: k8s/cert-manager/crds/kustomization.yaml [hunk 2]
```diff
@@ -100,82 +116,27 @@ patches:
       - op: add
         path: /metadata/labels
```
[the remark this answers is not in the document]
Comment (simardeep1792) [reply k8s/cert-manager/crds/kustomization.yaml]: Committed SHA (9f28808865e47c2866c3b643c1a902bf33bf4dcd)

File: k8s/cert-manager/crds/kustomization.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (simardeep1792) [reply k8s/cert-manager/crds/kustomization.yaml]: Committed SHA (9f28808865e47c2866c3b643c1a902bf33bf4dcd)

File: k8s/cert-manager/crds/kustomization.yaml [hunk 3]
```diff
@@ -11,59 +11,75 @@ patches:
...
+        path: /spec/template/spec/containers/0/resources
         value:
-          - --v=2
-          - --leader-election-namespace=cert-manager
+          requests:
+            cpu: 250m
+            memory: 512Mi
   - target:
       kind: Role
       name: cert-manager:leaderelection
```
[the remark this answers is not in the document]
Comment (simardeep1792) [reply k8s/cert-manager/crds/kustomization.yaml]: Committed SHA (9f28808865e47c2866c3b643c1a902bf33bf4dcd)

**Review by vedantthapa — APPROVED** [review]
Thanks!

**Comment by vedantthapa** [comment]
Relates to #155

**Pull request merged**